tools/reinstall-node-modules.js

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env node
+const {exec} = require('child_process');
+const path = require('path');
+
+const isWindows = process.platform === 'win32';
+const command = path.join(__dirname, `reinstall-node-modules${isWindows ? '.cmd' : ''}`);
```

Comment:
can you add a '.' in front here.

Reply:
@robgev `const command = '.' + path.join(__dirname, `reinstall-node-modules${isWindows ? '.cmd' : ''}`);`

Reply:
this __dirname is the absolute address.
it will be something like /home/<username>/Documents/zulip-electron/tools/

Reply:
I know that, but can you add it either way to avoid further problems related to this. Thanks!

Reply:
@cPhost it will not work that way. it will start look at ./home/username, which means home folder inside the current folder.
This is a wrong correction.

Reply:
Cool! Maybe I was pre-thinking  an issue too much.